region/norfair/crocomire.json

```diff
@@ -1294,7 +1294,7 @@
               "id": 4,
               "strats": [
                 {
-                  "name": "PCJR Bluespace",
+                  "name": "PCJR Bluespace Speed Blocks from the Left",
```

Comment:
I think the alt name is `Space Boost`, but I I'd like to see something like `PCJR Full Room Space Jump Through Speed Blocks`.
Its also not 100% clear if this is from the door or the acid from the name as `BlueSpaceJump` does work for both.